src/Microsoft.Health.Fhir.Api/Features/Audit/AuditLogger.cs

```diff
@@ -31,7 +31,8 @@ public class AuditLogger : IAuditLogger
             "Action: {Action}" + Environment.NewLine +
             "StatusCode: {StatusCode}" + Environment.NewLine +
             "CorrelationId: {CorrelationId}" + Environment.NewLine +
-            "Claims: {Claims}";
+            "Claims: {Claims}" + Environment.NewLine +
+            "CustomHeaderValues: {CustomHeaderValues}";
```

Comment:
I think to be on the safe side, we probably don't want to output this here in the default logger just like how we don't output IP address in this implementation.